selection-grid-flow/src/main/resources/META-INF/resources/frontend/src/helpers.js

```diff
@@ -35,12 +35,31 @@ export function _selectionGridSelectRow(e) {
         }
     }
 }
+export function _debounce(func, wait, immediate) {
+	debugger;
```

Comment:
Were the debugger statements left intentionally?

Reply:
Good catch, probably not, will fix this right away. Thanks!